CONTRIBUTING.md

```diff
@@ -48,12 +48,12 @@ We strive to follow this [recommendation](https://chris.beams.io/posts/git-commi
 - [Capitalize the subject line](https://chris.beams.io/posts/git-commit/#capitalize).
 - [Do not end the subject line with a period](https://chris.beams.io/posts/git-commit/#end).
 - [Use the imperative mood in the subject line](https://chris.beams.io/posts/git-commit/#imperative).
-- [Wrap the body at 72 characters](https://chris.beams.io/posts/git-commit/#wrap-72).
```

Comment:
We should explain why we don't follow this rule even though it is part of the reference we "strive to follow":

"As we might use URLs for references in commit messages, we do not wrap the body text to ensure links are clickable in Git user interfaces."